scripts/#5 Latest order.sql

```diff
@@ -0,0 +1,5 @@
+SELECT users.firstname, users.lastname,orders.orderDateTime
+FROM users LEFT JOIN orders

```

Comment:
`INNER JOIN` could be better because this way you eliminate users with no orders.